source/jormungandr/jormungandr/scenarios/tests/new_default_tests.py

```diff
@@ -343,48 +343,62 @@ def add_pt_sections(journey):
 def get_kraken_calls_test():
     for md in ["bike", "walking", "bss", "car", "ridesharing", "taxi"]:
         req = {"origin_mode": [md], "destination_mode": [md]}
-        assert get_kraken_calls(req) == {(md, md)}
+        assert get_kraken_calls(req) == {(md, md, "indifferent")}
 
-    req = {"origin_mode": ["bss", "walking"], "destination_mode": ["walking"]}
-    assert get_kraken_calls(req) == {("walking", "walking")}
+    req = {"origin_mode": ["bss", "walking"], "destination_mode": ["walking"], "direct_path": "none"}
+    assert get_kraken_calls(req) == {("walking", "walking", "none")}
 
-    req = {"origin_mode": ["bike", "walking"], "destination_mode": ["walking"]}
-    assert get_kraken_calls(req) == {("walking", "walking"), ("bike", "walking")}
+    req = {"origin_mode": ["bike", "walking"], "destination_mode": ["walking"], "direct_path": "only"}
+    assert get_kraken_calls(req) == {("walking", "walking", "only"), ("bike", "walking", "only")}
```

Comment:
Not sure, it would be clearer to have ` {("walking", "walking", "only"), ("bike", "bike", "only")}` in that case.
It can bloat the code, though.